core/ibft.go

```diff
@@ -747,15 +748,14 @@ func (i *IBFT) runPrepare(ctx context.Context) error {
 		// Grab the current view
 		view = i.state.getView()
 
-		// Grab quorum information
-		quorum = i.backend.Quorum(view.Height)
-
 		// Subscribe to PREPARE messages
 		sub = i.messages.Subscribe(
 			messages.SubscriptionDetails{
-				MessageType:    proto.MessageType_PREPARE,
-				View:           view,
-				MinNumMessages: int(quorum) - 1,
+				MessageType: proto.MessageType_PREPARE,
+				View:        view,
+				HasQuorumFn: func(view *proto.View, messages []*proto.Message) bool {
+					return len(messages)-1 >= 1
```

Comment:
Why is this condition not `i.backend.HasQuorum(messages) - 1 > 0`?

Reply:
Fixed in 83353bef1c07c08cb5fea7f4302d64bcd449e870